packages/playwright/src/PageExtensions.cs

```diff
@@ -70,5 +80,46 @@ private static async Task<AxeResult> RunAxeInner(this IPage page, AxeRunContext?
 
             return results;
         }
+
+        /// <summary>
+        /// Runs Axe against the page in its current state.
+        /// </summary>
+        /// <param name="page">The Playwright Page object</param>
+        /// <param name="context">Context to specify which element to run axe on.</param>
+        /// <param name="options">Options for running Axe.</param>
+        /// <param name="axeSource">Source code for axe-core</param>
+        /// <returns>The AxeResult</returns>
+        [Obsolete("Legacy Mode is being removed in the future. Use with caution!")]
+        public static async Task<AxeResult> RunAxeLegacy(
```

Comment:
This should be documented in the README's API docs section (with the same warning as the corresponding section of [@axe-core/playwright's setLegacyMode docs](https://github.com/dequelabs/axe-core-npm/tree/develop/packages/playwright#axebuildersetlegacymodelegacymode-boolean--true))

Reply:
(see #68 for what I think this should probably look like)